src/ClientWrapper.php

```diff
@@ -42,6 +42,9 @@ public function getClientForBranch(string $branchId): BranchAwareClient
             throw new ClientException(sprintf('Branch ID must be a number. "%s" given.', $branchId));
         }
         $this->resolveBranches();
+        if (!isset($this->branches[$branchId])) {
+            throw new ClientException(sprintf('Branch with ID "%s" does not exists.', $branchId));
```

Comment:
aby nám to v IM neházelo pak app error

Reply:
zatr sakr, žil jsem v tom, že BranchAwareClient to validuje

Reply:
```suggestion
            throw new ClientException(sprintf('Branch with ID "%s" does not exist.', $branchId));
```

Reply:
Nejde o `BranchAwareClient` tady ti to skonci uz v tom wrapperu ze zapisujes property do null